devops/Makefile

```diff
@@ -328,14 +266,14 @@ build-web:
 
 init-patroni:
 	@echo "++\n Init Patroni in $(NAMESPACE) \n++"
-	@oc -n $(NAMESPACE) process -f openshift/patroni-pre-req.yml | oc -n $(NAMESPACE) apply -f -
+	@oc -n $(NAMESPACE) process -f openshift/database/patroni-pre-req.yml | oc -n $(NAMESPACE) apply -f -
```

Comment:
Should `Redis` be part of this "database" folder also?

Reply:
yes it should be, I will move it. When I was developing Dheepak code was not merged, will take care.

Reply:
I believe that Patroni secrets and Mongo are also in the database folder. Can the Redis also me moved?